share/lib/python/neuron/__init__.py

```diff
@@ -602,7 +602,7 @@ def nrn_dll_sym_nt(name, type):
         if h.nrnversion(8).find("i686") == 0:
             b = "bin"
         path = os.path.join(h.neuronhome().replace("/", "\\"), b)
-        fac = 10 if sys.version_info[1] < 10 else 100 # 3.9 is 39 ; 3.10 is 310
+        fac = 10 if sys.version_info[1] < 10 else 100  # 3.9 is 39 ; 3.10 is 310
```

Comment:
I don't particularly like that this change is here, but somehow we ended up with Python code that wasn't consistent with `black`, so we might as well fix it now.